.github/workflows/_charm-release.yaml

```diff
@@ -29,6 +29,12 @@ on:
         required: false
         description: |
           The snap channel from which to install Charmcraft.
+      release-channel:
+        type: string
+        default: latest/edge
```

Comment:
are we sure about this default channel?

Reply:
We are; we currently release stuff to `latest/edge` when we merge to main